pyproject.toml

```
@@ -48,18 +46,51 @@ text = "BSD 3-Clause License"
 Homepage = "https://jupyter.org"
```

Let's add a link to the repo

```suggestion
Homepage = "https://jupyter.org"
Source = "https://github.com/jupyter-server/jupyter_ydoc"
```